src/main/java/uk/nhs/hee/tis/revalidation/connection/service/ElasticSearchIndexUpdateHelper.java

```diff
@@ -180,7 +183,9 @@ private boolean isException(final ConnectionInfoDto connectionInfo) {
       isExpired = (connectionInfo.getProgrammeMembershipEndDate().isBefore(LocalDate.now())
           && connectionInfo.getConnectionStatus().equalsIgnoreCase("Yes"));
     }
-    return (isVisitor || isExpired);
+    var isGmcFail = (connectionInfo.getExceptionReason() != null);
```

Comment:
Do we want to check for empty string here too?

Reply:
Thanks. I have used hasText() instead.